src/test/java/testsuite/integration/utility/AuroraTestUtility.java

```diff
@@ -182,7 +186,8 @@ public String createCluster() throws InterruptedException {
             .withSourceRegion(dbRegion)
             .withEnableIAMDatabaseAuthentication(true)
             .withEngine(dbEngine)
-            .withStorageEncrypted(true);
+            .withStorageEncrypted(true)
+            .withTags(testRunnerTag);
```

Comment:
What is it for?

Reply:
Setting up for AWS Maintenance Windows to automatically delete the clusters if for some reason, the tests did not delete them. I've noticed it happen twice and this is just another safety net / to catch if it fails.